build.gradle

```diff
@@ -55,3 +48,11 @@ if (JavaVersion.current().isJava8Compatible()) {
         }
     }
 }
+
+apply plugin: "io.codearte.nexus-staging"
+
+nexusStaging {
+    username = project.hasProperty('NEXUS_USERNAME') ? NEXUS_USERNAME : ""
```

Comment:
should we update deploy.gradle?

https://github.com/stripe/stripe-android/blob/d3c4df4ecf3e584732b5a44921c802612053740b/stripe/deploy.gradle#L26

Reply:
To be clear, we need the Nexus credentials for two different things:
- publishing the artifacts to Sonatype, which is done via `deploy.gradle` in the `stripe` subproject
- closing and releasing the staging repo on Sonatype, which is done via the `nexus-staging` plugin in the root project

I think the first part works correctly as it is, so I'd rather not modify `deploy.gradle` if it's not necessary.

(I couldn't find any clear documentation on how Gradle properties are supposed to work in general or with sub-projects or why I had to use `project.hasProperty` in the root project instead of just `hasProperty`. This fix is basically the result of me throwing code at the wall until something stuck.)